web/app/models/DrivingSchool.js

```diff
@@ -0,0 +1,17 @@
+// grab the mongoose module
+var mongoose = require("mongoose");
+
+// define our driving school model
+// module.exports allows us to pass this to other files when it is called
+var DrivingSchoolSchema = new mongoose.Schema({
+    schoolId: { type: Number, required: true },
+    owners: [{ type: Schema.Types.ObjectId, ref: "User" }],
+    instructors: [{ type: Schema.Types.ObjectId, ref: "User" }],
+    students: [{ type: Schema.Types.ObjectId, ref: "User" }],
```

Comment:
Just to make sure, @DylanWalseth, this is valid for referencing other documents? 
http://mongoosejs.com/docs/populate.html